core/templates/services/angular-services.index.ts

```diff
@@ -310,6 +311,7 @@ import { TranslationLanguageService } from 'pages/exploration-editor-page/transl
 import { UtilsService } from 'services/utils.service';
 import { ValidatorsService } from 'services/validators.service';
 import { PromoBarBackendApiService } from 'services/promo-bar-backend-api.service';
+import { UndoRedoService } from 'domain/editor/undo_redo/undo-redo.service';
```

Comment:
Can you move this to its right position?

Reply:
corrected.